math/polynomial_interpolation_on_geometric_sequence/info.toml

```
@@ -0,0 +1,35 @@
+title = 'Polynomial Interpolation (Geometric Sequence)'
+timelimit = 10.0
```

```suggestion
timelimit = 5.0
```
標準が 5 sec らしいので特に伸ばさなくてもいいかなという気がしました。